### PR TITLE
Bug/94283 Selected scope is removed when making a new search

### PR DESCRIPTION
**Pull request by Prestholm**
# Description

Please include a summary of the change and which issue is fixed. 
List any external dependencies that are required for this change.

Completes: [AB#94283](https://statoildeveloper.visualstudio.com/b6d97460-2e53-47ae-aad4-b41396d90828/_workitems/edit/94283)

# Checklist:

- [x] I have performed a self-review of my own code.
- [x] I have linked my DevOps task using the AB# tag.
- [x] My code is easy to read, and comments are added where needed.
- [ ] My code is covered by tests.

### CODE REVIEW

**Review by VRiveland — APPROVED** [review]

File: src/modules/InvitationForPunchOut/views/CreateAndEditIPO/SelectScope/CommPkgTable.tsx [hunk 1]
```diff
@@ -308,7 +295,6 @@ const CommPkgTable = forwardRef(
             rowData: CommPkgRow[],
             _row: CommPkgRow
         ): void => {
-            removeAllSelectedCommPkgsInScope();
```
Comment (bratli) [inline src/modules/InvitationForPunchOut/views/CreateAndEditIPO/SelectScope/CommPkgTable.tsx:311]: I am having trouble getting frontend locally, but have you tested than unselecting packages works as expected?

Comment (VRiveland) [reply src/modules/InvitationForPunchOut/views/CreateAndEditIPO/SelectScope/CommPkgTable.tsx:311]: Woah, I hadn't tested that properly at least. Unselecting all doesn't work.

Comment (bratli) [reply src/modules/InvitationForPunchOut/views/CreateAndEditIPO/SelectScope/CommPkgTable.tsx:311]: :(

**Review by bratli — CHANGES_REQUESTED** [review]

File: src/modules/InvitationForPunchOut/views/CreateAndEditIPO/SelectScope/CommPkgTable.tsx (hunk 1 above)
Comment (bratli) [reply src/modules/InvitationForPunchOut/views/CreateAndEditIPO/SelectScope/CommPkgTable.tsx:311]: Need to fix this

**Review by bratli — CHANGES_REQUESTED** [review]

File: src/modules/InvitationForPunchOut/views/CreateAndEditIPO/SelectScope/CommPkgTable.tsx (hunk 1 above)
Comment (Prestholm) [reply src/modules/InvitationForPunchOut/views/CreateAndEditIPO/SelectScope/CommPkgTable.tsx]: Nice catch!

**Review by bratli — APPROVED** [review]
Don't unserstand why this works, but seems fine when testing

🥇

**Review by VRiveland — APPROVED** [review]

**Pull request merged**